src/main/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/service/SmokeTestService.kt

```diff
@@ -11,19 +10,14 @@ class SmokeTestService(
     private val communityService: CommunityService,
 ) {
 
-  fun runSmokeTest(testMode: TestMode): Flux<TestResult> {
-    val probationDataResetResult = communityService.resetTestData("X360040");
-    if (probationDataResetResult.outcome == false) return Flux.just(probationDataResetResult)
-
-    val triggerTestResult = prisonService.triggerTest("A7742DY")
-    if (triggerTestResult.outcome == false) return Flux.fromIterable(listOf(probationDataResetResult, triggerTestResult))
+  fun runSmokeTest() :Flux<TestResult> {
 
     return Flux.concat(
-        Flux.just(probationDataResetResult),
-        Flux.just(triggerTestResult),
+        Flux.from(communityService.resetTestData("X360040")),
+        Flux.from(prisonService.triggerTest("A7742DY")),
         communityService.checkTestResults("A7742DY", "38479A"),
         Flux.from(communityService.checkTestResult("A7742DY", "38479A")).map { it.testResult }
```

Comment:
The split between these 2 lines should be:
checkTestResults (rename to wait test finish?) - indicates that the test is finished and we have data to check
checkTestResult - checks that the actual data is correct in nomins and delius:
* correct prison
* check key dates are same
* community endpoint /secure/offenders/nomsNumber/A742DY/custody/bookingNumber/38379A has all info
* prison endpoints /api/prisoners/A7742DY & /api/bookings/{{latestBookingId}}/sentenceDetail